src/main/java/org/jboss/pull/player/Main.java

```diff
@@ -5,6 +5,8 @@
  */
 package org.jboss.pull.player;
 
+import org.jboss.dmr.ModelNode;
```

Comment:
This looks to be cruft.

Reply:
+1 will do a sweep for these as well. Note I'm pretty much done with testing this (I tested on jboss-openshift-modules, so we can push it out and keep an eye on it in the morning.)